scrambler.cpp

```diff
@@ -863,7 +868,13 @@ void usage(const char *program)
               << "    -support-z3 [true|false]\n"
               << "        controls whether to support non-SMTLIB commands that "
                  "are supported\n"
-              << "        by Z3 (default: false)\n";
+              << "        by Z3 (default: false)\n"
+              << "        controls whether the output is in a format suitable for the model\n"
+              << "        track of SMT-COMP (default: false)\n"
+              << "\n"
+              << "    -count-asrts [true|false]\n"
```

Comment:
can you just write "-count-asserts"? It's more clear and not much longer

Reply:
Should be done now.